spec/forms/hub/outbound_call_form_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+describe Hub::OutboundCallForm do
+  let(:user) { create :admin_user, phone_number: "+18324658840" }
+  let(:client) { create :client, intake: (create :intake, phone_number: "+18324651680") }
+  context "initialization" do
+    context 'with client and user, no custom params' do
+      subject { described_class.new(client: client, user: user) }
+
+      it 'sets client_phone_number from client and user_phone_number from user' do
+        expect(subject.user_phone_number).to eq user.phone_number
+        expect(subject.client_phone_number).to eq client.phone_number
+      end
+    end
+
+    context "with custom parameters" do
+      subject do
+        described_class.new(
+            { user_phone_number: user_phone_number, client_phone_number: client_phone_number},
+            client: client, user: user)
+      end
+
+      let(:user_phone_number) { "+18324657780" }
+      let(:client_phone_number) { "+18324650000" }
+
+      it "sets client_phone_number and user_phone_number from params" do
+        expect(subject.user_phone_number).to eq user_phone_number
+        expect(subject.client_phone_number).to eq client_phone_number
+      end
+    end
+  end
+
+  context "call!" do
+    subject { described_class.new(client: client, user: user) }
+    let(:twilio_double) { double(Twilio::REST::Client) }
+    let(:twilio_calls_double) { double }
+    let(:twilio_response_double) { double(Twilio::REST::Api::V2010::AccountContext::CallInstance, sid: "123456", status: "initiated") }
+
+    before do
+      allow(Twilio::REST::Client).to receive(:new).and_return(twilio_double)
+      allow(twilio_double).to receive(:calls).and_return(twilio_calls_double)
+      allow(twilio_calls_double).to receive(:create).and_return(twilio_response_double)
+      allow(EnvironmentCredentials).to receive(:dig).with(:twilio, :voice_phone_number).and_return '+14156393361'
+    end
+
+    it "initializes a twilio instance" do
+      subject.call!
+      expect(Twilio::REST::Client).to have_received(:new).with(EnvironmentCredentials.dig(:twilio, :account_sid), EnvironmentCredentials.dig(:twilio, :auth_token))
```

Comment:
Possibly mock out the environment credentials to receive real fake numbers (likely nil right now)